src/Task/Archive/Pack.php

```diff
@@ -201,16 +201,39 @@ public function run()
      */
     protected function archiveTar($archiveFile, $items)
```

Comment:
@kporras07 would love a second set of eyes on this rewrite. The Archive_Tar has security issues so we need to replace it with PHP's PharData class. Tests on the archives are failing ATM.

Reply:
It looks good. I think $p_remove_dir and $p_add_dir are now dangling over there unused.

Other than that, I think it looks good but I haven't tested it.

Let me know if you need me to test it somehow or do anything else with this

Reply:
If there is a general need to remove archive_tar, perhaps that should happen in a separate PR.

Reply:
Its more a need to get tests to pass so the workflows work. But archive_tar is on the list of security issues, but that's incidental to making the Pack and Extract tests pass.

Reply:
Anyway, found there was a patched version and updated to the patched version and restored the original code.